### DISCUSSION TITLE
How to make ggplot2 inline?

### DISCUSSION THREAD

**Discussion by trafficfan** [posit-dev/positron#4063] [Q&A] [answered]
I've tried a lot , still my plot can't show inline.  But the notebook I've downloaded from kaggle worked well , Is it a bug or feature?
![image](https://github.com/user-attachments/assets/0d5fcd5a-5b07-4b7e-8466-0fc767b221c3)

**Comment by jmcphers** [comment]
It's a bug! Follow https://github.com/posit-dev/positron/issues/3846 for updates.

**Reply by trafficfan** [reply]
I noticed that bug was filed two weeks ago, and the version I'm using is two days old... has this bug been fixed now?

**Reply by jmcphers** [reply, marked as answer]
It has not; the issue will be closed when we fix the bug.
